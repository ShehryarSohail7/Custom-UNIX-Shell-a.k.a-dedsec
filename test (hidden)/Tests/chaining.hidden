echo "ls" > command.txt && echo "exit" >> command.txt && ../build/Shell < command.txt ; rm command.txt
echo "S" && echo "u" || echo "u" && echo "n" && echo "d" ; echo "a" && echo "y"
echo "M" && echo "o" || echo "o" && echo "n" && echo "d" ; echo "a" && echo "y"
echo "T" && echo "u" || echo "u" && echo "e" && echo "s" && echo "d" ; echo "a" && echo "y"
echo "W" && echo "e" || echo "e" && echo "d" && echo "n" && echo "e" && echo "s" && echo "d" ; echo "a" && echo "y"
echo "T" && echo "h" || echo "h" && echo "u" && echo "r" && echo "s" && echo "d" ; echo "a" && echo "y"
echo "F" && echo "r" || echo "r" && echo "i" && echo "d" ; echo "a" && echo "y"
echo "S" && echo "a" || echo "a" && echo "t" && echo "u" && echo "r" && echo "d" ; echo "a" && echo "y"