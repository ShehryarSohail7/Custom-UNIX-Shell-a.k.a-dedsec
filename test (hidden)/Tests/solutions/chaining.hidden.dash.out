
We are DedSec!! 
ls
&&
*
;
Tests
command.txt
config.json
cp
find.log
ls
names.txt
test.py
test_dir
exit
Exiting the shell...
S
u
n
d
a
y
M
o
n
d
a
y
T
u
e
s
d
a
y
W
e
d
n
e
s
d
a
y
T
h
u
r
s
d
a
y
F
r
i
d
a
y
S
a
t
u
r
d
a
y
